src/components/TradeWidget/index.tsx

```diff
@@ -685,7 +685,8 @@ const TradeWidget: React.FC<TradeWidgetProps> = ({
             sellToken={sellToken}
```

Comment:
Logic seems correct, but this component already receives the "sell token" and the "quote token", I understand this logic responsibility lies inside the `<Price Suggestions>`, isn't it?

I'm referring to:
```js
            limitPrice={sellToken === quoteToken ? priceValue : priceInverseValue}
```

Reply:
EDIT: Maybe you are right and the responsability is here?

My thinking now is, the one ignoring the quoted token to get a price is `TradeToken`, so this is the one that gets a price in a different quote to the one user wants to see it in.
Therefore, he makes sure that ask the price always as BUY/SELL, but if SELL is not your your quote, then its his responsibility to invert it.

Reply:
ok @anxolin did you mean like this https://github.com/gnosis/dex-react/pull/1557/commits/5eb283464db14a1f919b1c3c9753e84ee61dc3fc

Reply:
I was thinking aloud were the responsibility for flipping the price was.
I just didn't want to go back to flipping logic mess. So I did my first comment saying it should be moved to the suggestion

then I edited, and amended my comment, thinking that u may be right, and I explained why I thought so.

So as a summary, I'm not 100% sure, but my current thinking is that u had it right, this is why I approved

Reply:
ah ok, no worries thx